### ISSUE TITLE
HealthCheck route returns database error

### ISSUE THREAD

**Issue by galenguyer** [ComputerScienceHouse/profiles#114]
The healthcheck route returns the following json and a 513 status code even though profiles is functioning fine.

 ```json
{
  "database": {
    "server": null,
    "status": false
  },
  "ldap": {
    "server": "ldaps://ipa01-ih.csh.rit.edu",
    "status": true
  }
}
```

**Comment by adamhb123** [comment]
Yeah, the healthcheck implementation was never really properly finished iirc. I will look into this over the coming weeks to see about getting everything working properly.

**Comment by galenguyer** [comment]
closed by #116
